dotnet/src/SemanticKernel/CoreSkills/TextMemorySkill.cs

```diff
@@ -1,5 +1,6 @@
 ﻿// Copyright (c) Microsoft. All rights reserved.
 
```

Comment:
nit: It'd be cool to add UTs for this skill.

Reply:
unit tests* -- I know it might be difficult with things now due to the coupling in the http layer. hopefully that will change.

Even then, maybe an integration test for this stuff?

Reply:
Assigned a task to myself to create integration tests for textmemoryskill